src/Particle/LongRange/LRCoulombSingleton.h

```diff
@@ -72,7 +72,7 @@ struct LRCoulombSingleton
    */
   static std::unique_ptr<RadFunctorType> createSpline4RbyVs(const LRHandlerType* aLR,
                                                             mRealType rcut,
-                                                            const GridType* agrid = nullptr);
```

Comment:
Since now agrid is mandatory. I think the argument is better as "const GridType& agrid".